content/blog/2019/10/2019-10-30-jenkins-performance-avoiding-pitfalls.adoc

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Jenkins Performance: Avoiding Pitfalls, Diagnosing Issues, and Scaling for Growth - DevOps World | Jenkins World 2019"
+tags:
+- jenkins world
+- jenkinsworld
+- devopsworld2019
+- performance
+authors:
+- ryansmith303
+opengraph:
+  image: /images/performance/gc-unhealthy.png
+  image: /images/logos/jenkenstein/256.png
+---
+
+NOTE: This is a speaker blogpost for a DevOps World | Jenkins World 2019 talk in Lisbon, Portugal
+
+With Halloween upon us, there's no better time to discuss Jenkinstein! Are you suffering from Jenkins Performance issues? Are users complaining about a slow UI or even scarier, Jenkins is frequently going down?  
+
+image::/images/logos/jenkinstein/256.png[]
+
+Come join me at link:https://www.cloudbees.com/devops-world/lisbon[DevOps World | Jenkins World 2019] for "link:https://sched.co/UVVp[**Jenkins Performance: Avoiding Pitfalls, Diagnosing Issues, and Scaling for Growth**]", a talk about JVM administration, and best practices from the front lines of supporting thousands of Jenkins installations worldwide. During this talk we'll cover how to prevent your Jenkins instance from becoming a Jenkinstein! 
```

Comment:
One of the grammar engines I use has been suggesting that I should not use a separating comma when `and` is referring to exactly two things.  I'm not sure if this is a case for the Oxford comma or not, so I include it here as an entirely optional comment.  You're absolutely welcome to ignore this comment..
```suggestion
Come join me at link:https://www.cloudbees.com/devops-world/lisbon[DevOps World | Jenkins World 2019] for "link:https://sched.co/UVVp[**Jenkins Performance: Avoiding Pitfalls, Diagnosing Issues, and Scaling for Growth**]", a talk about JVM administration and best practices from the front lines of supporting thousands of Jenkins installations worldwide.
During this talk we'll cover how to prevent your Jenkins instance from becoming a Jenkinstein!
```

Reply:
Agree with this one. It's not about Oxford comma usage, just plain old comma usage :) If you were reading this aloud, you would not really pause before the second part of the sentence.